CHANGELOG.md

```diff
@@ -4,6 +4,10 @@ Todos los cambios notables a este proyecto serán documentados en este archivo.
 El formato está basado en [Keep a Changelog](http://keepachangelog.com/en/1.0.0/)
 y este proyecto adhiere a [Semantic Versioning](http://semver.org/spec/v2.0.0.html).
 
+## [1.0.3] - 2018-11-15
+### Changed
+- Actualiza sdk js a versión 1.5.8, que mejora el comportamiento para usuarios iOS que no poseen la aplicación Onepay instalada
```

Comment:
The merchant probably doesn't know what "js" is, nor why that version is relevant. Simplify it to just "Mejora el comportamiento para usuarios iOS que no poseen la aplicación Onepay instalada".

Reply:
@leosoto Okey, fixed in a905f30